data/human/free worlds 0 prologue.txt

```diff
@@ -1068,6 +1068,30 @@ mission "FW Bounty 1"
 
 
 
+ship "Behemoth" "Behemoth (Moonless Night)"
+	sprite "ship/behemoth moonless"
+	thumbnail "thumbnail/behemoth moonless"
+	add attributes
+		"hull" 300
+		"mass" 40
+		"drag" .2
+	outfits
+		"Sidewinder Missile Launcher"
```

Comment:
```suggestion
		"Sidewinder Missile Launcher" 2
```

Reply:
The Speedy variant this is a copy of seems to deliberately have one launcher (note just 45 ammo not 90 like ones with two launchers). Haven't checked but they probably can't fit it.

Reply:
No way it can fit this, it already has to use 6 outfit expansions

Reply:
Ah okay, if the variant only had one launcher, then this is fine too. I almost want to say, if we are worried about difficulty, you could even take off one of the Heavy Laser turrets so the build is more asymmetrical and janky like this ship comes off as ?

Reply:
It's an optional set of missions that only trigger if you have battle experience I think that we should wait for an issue saying if its too strong/weak before doing the appropriate changes

These missions felt too easy to me and like they should adapt to the player's strength overall but anyway I'm getting carried away